src/encoded/static/components/dataset.js

```diff
@@ -2270,6 +2271,84 @@ const generateSupplementalShortLabels = (experimentList, getSupplementalShortLab
 );
 
 
+/**
+ * Defines the number of experiments/control experiments that appear at once in the series dataset
+ * table.
+ */
+const PAGE_DATASET_COUNT = 25;
+
+
+/**
+ * Display one of the two sections (experiments or control experiments) of the dataset table on the
+ * Series pages.
+ */
+const SeriesDatasetTableSection = ({ title, datasets, tableColumns, sortColumn, adminUser, accessLevel }) => {
+    /** The current page of datasets if more than the max number of displayable datasets exist */
+    const [currentDatasetPage, setCurrentDatasetPage] = React.useState(0);
+
+    /**
+     * Called when the user clicks the pager to go to a new page of datasets.
+     * @param {number} New current page number of datasets to display
+     */
+    const updateCurrentExperimentPage = (newPage) => {
+        setCurrentDatasetPage(newPage);
+    };
+
+    const totalDatasetPageCount = Math.floor(datasets.length / PAGE_DATASET_COUNT) + (datasets.length % PAGE_DATASET_COUNT !== 0 ? 1 : 0);
+    const pageStartIndex = currentDatasetPage * PAGE_DATASET_COUNT;
+    const visibleDatasets = datasets.slice(pageStartIndex, pageStartIndex + PAGE_DATASET_COUNT);
+
+    return (
+        <>
+            <div className="experiment-table__subheader">
+                <>{title}s</>
+                <div className="experiment-table__subheader-controls">
+                    <CartAddAllElements elements={datasets.map((dataset) => dataset['@id'])} />
+                    {totalDatasetPageCount > 1
+                        ? (
+                            <Pager.Simple
+                                total={totalDatasetPageCount}
+                                current={currentDatasetPage}
+                                updateCurrentPage={updateCurrentExperimentPage}
+                            />
+                        )
+                        : null
+                    }
+                </div>
+            </div>
+            <div className="table-counts">
+                {datasets.length} {title.toLowerCase()}{datasets.length === 1 ? '' : 's'}
```

Comment:
In the future we should consider adding the 's'-code to objectutil since it is used a lot.